process-compose.yaml

```diff
@@ -490,7 +490,6 @@ processes:
       failure_threshold: 100
 
   marketplace-reserve-builder:
-    disabled: true
```

Comment:
This (and removing `[marketplace]` in `docker-compose.yml`) will cause conflicts, as both services bind to `ESPRESSO_BUILDER_SERVER_PORT`.

Reply:
do we want to run both builders at the same time or only one of those?

Reply:
fallback has a separate port in .env file?

Reply:
We want to run either both marketplace builders when we're upgraded to marketplace or one legacy builder otherwise

Reply:
ok I will revert it for now